pkg/controllers/mcs/mcs_svc_controller.go

```diff
@@ -0,0 +1,81 @@
+package mcs
+
+import (
+	"context"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/tools/record"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	mcsv1alpha1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+	"github.com/karmada-io/karmada/pkg/sharedcli/ratelimiterflag"
+	"github.com/karmada-io/karmada/pkg/util"
+)
+
+const (
+	// ServiceControllerName ServiceController name.
+	ServiceControllerName = "service-resourcebindings-controller"
+)
+
+// ServiceController lists/watches the resourcebindings resource,
+type ServiceController struct {
```

Comment:
Can we directly merge the tasks of this controller into the multi-cluster-service-controller?